run/GCClassic/HISTORY.rc.templates/HISTORY.rc.aerosol

```diff
@@ -292,7 +293,7 @@ COLLECTIONS: 'Restart',
 #==============================================================================
 # %%%%% THE SatDiagn COLLECTION %%%%%
 #
-# GEOS-Chem data during satellite overpass
+# GEOS-Chem data (on level edges) during satellite overpass
```

Comment:
Remove `(on level edges)` here

Reply:
@msulprizio: now fixed in commit d33d1b1